src/howard/howard/templates/howard/invoice.html

```diff
@@ -0,0 +1,80 @@
+{% load i18n %}
+{% load static %}
+
+<html>
+  {% if debug %}
+  <head>
+    <style>
+      {{ css }}
+    </style>
+  </head>
+  {% endif %}
+  <body>
+    <div class="invoice">
+      <header>
+        <div class="top">
+          <div class="label">{% trans "Invoice" %}</div>
+          <div class="logo">
+            <img
+              src="{{ debug | yesno:",file://" }}{% static "howard/logo-invoice.png" %}"
+              alt="{% trans "company logo" %}"
+            />
+          </div>
+        </div>
+        <div class="customer">
+          <div>
+            <div class="date">{% now "DATE_FORMAT" %}</div>
+            <div>
+              {{ order.customer.name }}<br>
+              {{ order.customer.address|safe }}
+            </div>
+          </div>
+        </div>
+      </header>
+      <article>
+        <div class="invoice-id">
+          {% trans "Invoice reference:" %} {{ order.invoice_id }}
+        </div>
+        <table>
+          <thead>
+            <tr>
+              <th>{% trans "Product" %}</th>
+              <th>{% trans "Description" %}</th>
+              <th>{% trans "Taxes" %}</th>
+              <th class="align-right">{% trans "Price" %}</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><b>{{ order.product.name }}</b></td>
+              <td>{{ order.product.description }}</td>
+              <td>{{ order.product.vat }}%</td>
+              <td class="align-right">{{ order.product.total }}</td>
+            </tr>
+          </tbody>
+        </table>
+        <table class="price">
+          <tr>
+            <td>{% trans "Subtotal" %}</td>
+            <td class="align-right">{{ order.product.subtotal }}</td>
+          </tr>
```

Comment:
What about something more semantic like the following?

```suggestion
          <tr class="subtotal">
            <td>{% trans "Subtotal" %}</td>
            <td>{{ order.product.subtotal }}</td>
          </tr>
```

And adapt the CSS selector:

```css
.price > tr > td:nth-child(2) {
  align: right;
}
```